Modules/Challenges/test/Challenges.API.Tests.Performance.EndToEnd/src/get-test.ts

```diff
@@ -0,0 +1,74 @@
+import http from 'k6/http';
```

Comment:
Damn, my example was a bit misleading. I wanted you to only change the separator between the test name and the `test` postfix.
So in case of `post-with-auth-test.ts`, it should be `post-with-auth.test.ts`.